client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -25,13 +26,18 @@ class NameserversToggle extends PureComponent {
 				</span>
 
 				<form className="name-servers__toggle">
-					<ToggleControl
-						id="wp-nameservers"
-						name="wp-nameservers"
-						onChange={ this.handleToggle }
-						checked={ this.props.enabled }
-						value="active"
-					/>
+					<FormLabel>
```

Comment:
Seems like wrapping everything in a `<FormLabel />` won't do the job, because `ToggleControl` already includes a `<label />`. This means we'll end up with a `<label />` inside a `<label />`, which is not semantic and is invalid HTML (see https://html.spec.whatwg.org/multipage/forms.html#the-label-element:phrasing-content-2-3).